docs/source/developers/ROADMAP_0_3_lookback.md

```diff
@@ -0,0 +1,109 @@
+
+    > We've reduced the number of open [`bug`](https://github.com/napari/napari/labels/bug) labeled issues by more than 20% in the past two months from > 80 to < 60 and added some more screenshot and GUI tests, but test coverage of the GUI can still be improved.
+
+- **Improve support for annotation** of points [#858](https://github.com/napari/napari/issues/858), shapes [#177](https://github.com/napari/napari/issues/177), labels, and images [#209](https://github.com/napari/napari/issues/209) including text rendering [#600](https://github.com/napari/napari/pull/600).
+
+    > We've now added support to label points with text and somewhat improved color handling for our points, shapes, and labels layers, but have more work to do to standardize those improvements and reduce code duplication. 
+
+- Add **linked multi-canvas support** ([#760](https://github.com/napari/napari/issues/760)) to allow orthogonal views, or linked views of two datasets, for example to select keypoints for image alignment, or simultaneous 2D slices with 3D rendering.
+
+    > We have not done this work yet, and this work will be moved to our 0.4 series of releases.
+
+- Add **layer groups** [#970](https://github.com/napari/napari/issues/970), which allow operating on many layers simultaneously making the viewer easier to use for multispectral or multimodal data, or, in the context of multiple canvases, where one wants to assign different groups to different canvases.
+
+    > We have not done this work yet, and this work will be moved to our 0.4 series of releases.
+
+- Improve the **user interface and design** of the viewer to make it easier to use. A [napari design audit](https://github.com/napari/napari/issues/469) last year dramatically improved the usability of the viewer. We must continue to run through these periodically to ensure the UI is friendly to new users, particularly non-programmers. See the [`design` label](https://github.com/napari/napari/labels/design) for more information.
+
+    > We have conducted a product heuristics analysis to [identify design and usability issues](https://github.com/napari/product-heuristics-2020), and will now be working during the 0.4 series of releases to implement them.
+
+## Make a downloadable application with basic plugin management and persistent settings
+
+- Distribute **a bundled application for each major OS** [#496](https://github.com/napari/napari/pull/496) to allow scientists to use napari without requiring a Python development environment, which can be difficult to install and maintain. The bundle should contain the necessary machinery to add plugins [#1074](https://github.com/napari/napari/issues/1074).
+
+    > We have succeeded in providing a bundled app that you can install plugins into. We have more work to do around plugin management that will continue in our 0.4 series of releases.
```

Comment:
I don't know how to improve this but it sounds so matter of fact when it is indeed so awesome! 😂

Reply:
ok more added